apps/src/sites/studio/pages/devise/sessions/_login.js

```diff
@@ -2,10 +2,22 @@ import $ from 'jquery';
 
 import {EVENTS, PLATFORMS} from '@cdo/apps/metrics/AnalyticsConstants';
 import analyticsReporter from '@cdo/apps/metrics/AnalyticsReporter';
+import statsigReporter from '@cdo/apps/metrics/StatsigReporter';
 
 $(document).ready(() => {
   analyticsReporter.sendEvent(EVENTS.LOGIN_PAGE_VISITED, {}, PLATFORMS.STATSIG);
 
+  const isInSignupExperiment = statsigReporter.getIsInExperiment(
+    'new_sign_up_v1',
+    'showNewFlow',
+    false
+  );
+  const signupLink = document.getElementById('signup-link');
+
+  if (isInSignupExperiment) {
+    signupLink.href = 'https://studio.code.org/users/new_sign_up/account_type';
```

Comment:
Wonder if we should, in some of these cases, make these relative? I know it's hard in the shared implementations but this page is dashboard-specific

Reply:
Do you know if we have a CDO.studio equivalent in apps? I was looking for one but then stopped for time/simplicity. I can also do relative if it's preferred.

Reply:
I'd prefer relative for this file, but yeah, the other files would need too much work :/ Just thinking about if/when we bug bash this -- are we trying to bug bash on an adhoc or could we bug bash in prod?

Reply:
Updated this path to be relative! I was imagining an adhoc unless you can think of a reason prod would be preferred!

Reply:
adhoc is good! Just will be a slightly weird experience everywhere else without the relative paths. I think this is good, though!

Reply:
Ahh yes that's fair.